FirebaseCombineSwift/Tests/Unit/Firestore/AddDocumentTests.swift

```diff
@@ -0,0 +1,217 @@
+// Copyright 2021 Google LLC
```

Comment:
Let's remove uncommented test code for now and add it back when we've sorted out the testing story/